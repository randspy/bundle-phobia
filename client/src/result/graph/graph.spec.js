import React from 'react';
import { shallow } from 'enzyme';
import Graph from './graph';

describe('Graph', () => {
  test('should wire component', () => {
    const wrapper = shallow(<Graph bundles={[]} />);
    expect(wrapper.exists()).toBe(true);
  });

  it('should render correctly', () => {
    const wrapper = shallow(
      <Graph
        bundles={[
          {
            name: 'react',
            version: '14.5.3',
            minified: 20024,
            minifiedPercentage: '33%',
            gzip: 10000,
            gzipPercentage: '17%'
          }
        ]}
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
