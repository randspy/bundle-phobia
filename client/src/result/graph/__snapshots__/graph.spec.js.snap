// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Graph should render correctly 1`] = `
<div
  className="graph"
>
  <div
    key="14.5.3"
  >
    <div
      className="graph-bundle"
    >
      <div
        className="graph-version"
      >
        <div
          className="graph-version-text"
        >
          14.5.3
        </div>
      </div>
      <div
        className="graph-col"
      >
        <Tooltip
          animateFill={true}
          animation="shift"
          arrow={false}
          arrowSize="regular"
          className=""
          delay="500"
          disabled={false}
          distance={10}
          duration={375}
          followCursor={false}
          hideDelay={0}
          hideDuration={375}
          hideOnClick={true}
          html={null}
          inertia={false}
          interactive={false}
          interactiveBorder={2}
          multiple={false}
          offset={0}
          onHidden={[Function]}
          onHide={[Function]}
          onRequestClose={[Function]}
          onShow={[Function]}
          onShown={[Function]}
          popperOptions={Object {}}
          position="top"
          size="regular"
          sticky={false}
          stickyDuration={200}
          style={Object {}}
          theme="dark"
          title="Minified 19.6kB | Gzipped 9.8kB"
          touchHold={false}
          trigger="mouseenter focus"
          unmountHTMLWhenHide={false}
          zIndex={9999}
        >
          <div
            className="graph-minified"
            style={
              Object {
                "width": "33%",
              }
            }
          />
           
          <div
            className="graph-gzip"
            style={
              Object {
                "width": "17%",
              }
            }
          />
        </Tooltip>
      </div>
    </div>
  </div>
</div>
`;
